docs/theme/assets/tokens.css

```
@@ -0,0 +1,116 @@
+/* Lance Design System — tokens (Swiss editorial: white, ink, brand purple #625EFF).
+   Merged from the design project's ds/tokens set; dark theme mirrors the same
+   structure with paper rules on ink. */
+@import url("https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;500;600;700&family=IBM+Plex+Sans:wght@400;500;600&family=Roboto+Mono:wght@400;500;600&display=swap");
```

Stylelint (17.14.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;500;600;700&family=IBM+Plex+Sans:wght@400;500;600&family=Roboto+Mono:wght@400;500;600&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@400;500;600;700&family=IBM+Plex+Sans:wght@400;500;600&family=Roboto+Mono:wght@400;500;600&display=swap"" (import-notation)

(import-notation)